staging/src/k8s.io/apiserver/pkg/util/flowcontrol/config.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	fcrequest "k8s.io/apiserver/pkg/util/flowcontrol/request"
+	"sigs.k8s.io/yaml"
+)
+
+// PriorityAndFairnessConfig holds priority and fairness parameters.
+type PriorityAndFairnessConfig struct {
+	// WorkEstimatorConfig holds work estimator parameter values.
+	fcrequest.WorkEstimatorConfig `json:"workEstimatorConfig,omitempty"`
+}
+
+// NewConfig creates a new PriorityAndFairnessConfig with default values.
+func NewConfig() PriorityAndFairnessConfig {
```

Comment:
Let's call it DefaultConfig()

Reply:
Actually - let's maybe remove this file from this PR - let's make this PR only introduce the flowcontrolrequest.Config and use it across the codebase.

Reply:
Done.